chart/values.schema.json

```diff
@@ -7409,6 +7409,14 @@
                             ],
                             "default": null
                         },
+                        "gitSshKey": {
```

Comment:
```suggestion
                        "sshKey": {
```

I think maybe `sshKey` is a better name for this?

Reply:
changed 👍